hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -1192,8 +1198,10 @@ public StoreFileWriter createWriterInTmp(long maxKeyCount, Compression.Algorithm
       if (shouldCacheDataOnWrite) {
         writerCacheConf.enableCacheOnWrite();
         if (!cacheOnWriteLogged) {
-          LOG.info("For Store {} , cacheDataOnWrite is true, hence enabled cacheOnWrite for " +
-            "Index blocks and Bloom filter blocks", getColumnFamilyName());
+          LOG.info("For Store {} in region {}, " +
```

Comment:
No need of 'Store' or 'region'... its obvious that we're talking store and region. Having these works in there bulk up logging and make it harder to read/parse.

Reply:
> No need of 'Store' or 'region'... its obvious that we're talking store and region. Having these works in there bulk up logging and make it harder to read/parse.

Got it